website/api/manager/ManagerAPI.php

```diff
@@ -81,20 +88,29 @@ protected function task() {
                 );
             }
 
+            // don't give out any task if games have been stopped.
+            if($this->checkConfig("compState", "noGameTasks")) {
+                return array("type" => "notask");
```

Comment:
This isn't super pretty. I believe it will make the workers error within their overarching try-except block.